Tetris/src/tetris/GameSystem.java

```diff
@@ -36,7 +51,7 @@ public GameSystem(){
 	 * @param rotation
 	 */
 	public static void moveActiveTetromino(int x_direction, int y_direction, int rotation){
```

Comment:
Am I understanding correctly this will be called by the action listener with the appropriate params?